README.md

```diff
@@ -62,29 +69,39 @@ details.
    * `APP_SERVICE_PORT` (number, required): the port on which the App service
      should listen on the local machine
    * `POSTGRES_API_SERVICE_PORT` (number, required): the port on which the
-     PostgreSQL service should listen on the local machine
-   * `API_POSTGRES_USER` (string, required): username to be used for the
-     PostgreSQL connection (API)
-   * `API_POSTGRES_PASSWORD` (string, required): password to be used for the
-     PostgreSQL connection (API)
-   * `API_POSTGRES_DB` (string, required): name of the database to be used for
-     the PostgreSQL connection (API)
+     Docker PostgreSQL service should listen on the local machine
+   * API PostgreSQL configuration variables:
+     * `API_POSTGRES_HOST` (string, required): host of the database server to be
+       used for the PostgreSQL connection (API)
+     * `API_POSTGRES_PORT` (number, required): port of the database server to be
+       used for the PostgreSQL connection (API)
+     * `API_POSTGRES_USER` (string, required): username to be used for the
+       PostgreSQL connection (API)
+     * `API_POSTGRES_PASSWORD` (string, required): password to be used for the
+       PostgreSQL connection (API)
+     * `API_POSTGRES_DB` (string, required): name of the database to be used for
+       the PostgreSQL connection (API)
```

Comment:
[minor] it is always hard to maintain the .md files - for this very particular case, there is a common approach to provide `.env.example` tho for starters which document (and provides some defaults) the variables.